src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2810,34 +2810,24 @@ class misc_commandscript : public CommandScript
         }
 
         if (target->IsPlayer())
-        {
             if (handler->HasLowerSecurity(target->ToPlayer()))
-            {
                 return false;
-            }
-        }
 
-        if (!target->IsAlive())
-        {
+        if (!target->IsAlive() || !damage)
             return true;
-        }
-
-        if (!damage)
-        {
-            return true;
-        }
 
         if (target->GetTypeId() == TYPEID_UNIT && handler->GetSession()->GetSecurity() == SEC_CONSOLE) // pussywizard
-        {
             target->ToCreature()->LowerPlayerDamageReq(target->GetMaxHealth());
-        }
+
+        if (percent)
+            if (StringStartsWith("pct", *percent))
+                if (damage <= 100)
+                    damage = target->CountPctFromMaxHealth(damage);
```

Comment:
Probably add an else block here, too? Limit to 100 or something maybe

Reply:
No, it will just default to dealing the raw damage if it's higher than 100 percent

Reply:
Alright, I thought some kind of "error" handling would be nice here. So when you do 200pct damage it will either do 100pct or show an error instead of doing 200 damage, which would be a bit unexpected imo